park_api/cities/Oldenburg.py

```diff
@@ -22,7 +22,7 @@ def parse_html(html):
     # last_updated is the date when the data on the page was last updated
     last_updated = str(soup.select("body"))
     start = str.find(last_updated, "Letzte Aktualisierung:") + 23
-    last_updated = last_updated[start:start + 16] + ' Uhr'
+    last_updated = last_updated[start:start + 16]
```

Comment:
This change makes the tests fail. Either add Uhr or remove the ` Uhr` from https://github.com/offenesdresden/ParkAPI/blob/c1109ce942727a6c71293ab38e7f3e40120c3459/park_api/cities/Oldenburg.py#L30.

Reply:
done.

in my branch. Do I have to make a pull-request again?

Reply:
No, as you can see your fix has been added to this pull request and the tests work now.